logstash-core/src/main/java/org/logstash/secret/store/backend/JavaKeyStore.java

```diff
@@ -92,13 +98,13 @@ public JavaKeyStore(Path keyStorePath, char[] keyStorePass) {
                 String keyStorePermissions = System.getProperty("logstash.keystore.file.perms", "rw-rw----");
```

Comment:
I missed this last time, but in a future PR, let's get this out of system properties and put it in `logstash.yml` *or* make it not a setting at all (default to 0600 is what other tools like ssh-keygen, ruby netrc, etc default to)

Reply:
I defaulted to E.S.'s default [here](https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/common/settings/KeyStoreWrapper.java#L333).  It looks like [this](https://github.com/elastic/elasticsearch/commit/7a035f5f8435962072157f46031e73855fe4a875) is the reason.  I made a note to revisit this while doing package testing. 

> let's get this out of system properties and put it in logstash.yml or make it not a setting at all

It's here mostly for testing, and had not planned on documenting this. I have other ways I can set this for testing for you prefer to expose it at all.  I don't think it should be .yml setting. 

thoughts ?

Reply:
I'll go ahead and nuke the System property, but leave the 660.

Reply:
Ok I'll file a blocking issue on the permissions to follow up.

Reply:
https://github.com/elastic/logstash/issues/8735

Reply:
thanks for logging the follow up, we can continue the 600 vs. 660 conversation over there.